apps/chat/channels.py

```diff
@@ -136,7 +136,9 @@ def __init__(
         self.experiment_channel = experiment_channel
         self._experiment_session = experiment_session
         self._message: BaseMessage = None
-        self._participant_identifier = experiment_session.participant.identifier if experiment_session else None
+        participant = experiment_session.participant
+        self._participant_identifier = participant.identifier if experiment_session else None
+        self._participant_id = participant.id
```

Comment:
This logic doesn't quite work since `experiment_session` could be None here. I suggest following the same pattern as the `participant_user` property: https://github.com/dimagi/open-chat-studio/blob/64f6faecc5fcc43fafd54db52c65c14cb54ffca9/apps/chat/channels.py#L216-L219